README.md

```diff
@@ -1,36 +1,60 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15563518&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
-
+software engineering is a systematic approach to the design, development, operation, and maintenance of software. It involves applying engineering principles and practices to software development, with the goal of creating high-quality software products that meet the needs of users and stakeholders.
+- It is important in the technology industry because it helps ensure that software projects are completed on time, within budget, and to the required quality standards.
+- It improves the efficiency and effectiveness of the software development process, leading to better software products and services.
```

Comment:
1st Milestone: The introduction of structured programming in the 1960s.
2nd Milestone: The development of object-oriented programming in the 1980s.
3rd Milestone: The rise of Agile methodologies in the early 2000s.